docs/apache-airflow/core-concepts/operators.rst

```diff
@@ -185,10 +185,23 @@ This will fail with ``TemplateNotFound: cat script.sh``, but we can prevent airf
 
     fixed_print_script = BashOperator(
         task_id="fixed_print_script",
-        bash_callable="cat script.sh",
+        bash_command="cat script.sh",
     )
     fixed_print_script.template_ext = ()
```

Comment:
Let’s move the `literal` solution before this. We want to recommend it over this somewhat hacky workaround.